redis-pq-demo/README.md

```diff
@@ -0,0 +1,29 @@
+# Example run-time
+
+This is a demo that builds and deploys an instance of acapy that is configured to act as a mediator, and use REDIS to as a resilient method of managing acapy's event queue. 
+
+How to see a full fledged demo, running a local ledger, and a set of agents using the mediator (with redis)  
```

Comment:
Still nothing about how to get a local ledger going.

Reply:
the mediator is set to `no-ledger`, so i'm going to defer all of that to the agents the dev wants to run behind the mediator. good catch.